### PR TITLE
(6) feat: add network settings flow stack

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- feat: add NetworkSettingsFlowStack

>[!NOTE]
> I edited the commit that add NetworkSettingsFlowStack to remove the module alias.

Remove the commits related to:
- chore: add babel-plugin-module-resolver@5.0.0
- babel: configure @components and @screens module aliases

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/screens/NetworkSettings/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { NetworkSettingsFlowStack, NetworkSettingsFlowNav } from "./NetworkSettingsFlowStack";
+
+export {
+  NetworkSettingsFlowStack,
+  NetworkSettingsFlowNav
+}
```
Comment (andreabadesso) [inline src/screens/NetworkSettings/index.js:6]:
```suggestion
/**
 * Copyright (c) Hathor Labs and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

import { NetworkSettingsFlowStack, NetworkSettingsFlowNav } from './NetworkSettingsFlowStack';

export {
  NetworkSettingsFlowStack,
  NetworkSettingsFlowNav,
};
```

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: src/screens/NetworkSettings/NetworkSettingsFlowStack.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { createStackNavigator } from '@react-navigation/stack';
+
+export const NetworkSettingsFlowNav = Symbol('NetworkSettingsFlowStack').toString();
+
+export const NetworkSettingsFlowStack = ({ navigation }) => {
+  const FlowStack = createStackNavigator();
+  return (
+    <FlowStack.Navigator
+      initialRouteName='NetworkSetitngsDisclaimer'
```
Comment (pedroferreira1) [inline src/screens/NetworkSettings/NetworkSettingsFlowStack.js:9]:
```suggestion
      initialRouteName='NetworkSettingsDisclaimer'
```

File: src/screens/NetworkSettings/index.js (hunk 1 above)
Comment (alexruzenhack) [reply src/screens/NetworkSettings/index.js:13]: - Add copyright: abe71c9d5fa5412a11994ce8416712ac2419d046

File: src/screens/NetworkSettings/NetworkSettingsFlowStack.js (hunk 2 above)
Comment (alexruzenhack) [reply src/screens/NetworkSettings/NetworkSettingsFlowStack.js]: - fix typo: 35a5b887be07c2db091c84dfdc4ccbd132511129

**Review by andreabadesso — APPROVED** [review]

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**